### DISCUSSION TITLE
Can't run on my Dell G3 3579 Laptop

### DISCUSSION THREAD

**Discussion by kaderci-johannes** [sultim-t/xash-rt#70] [General]
I have a clean installation of Half Life 1 on my Windows 10 and I precisely followed the installation instructions. However, when I run the rt_bin/xash3d.exe, I get an error message saying "Host_InitError: Can't find physical device with ray tracing support" and the program doesn't run at all. My laptop has Nvidia graphics. What could be the reason?

**Comment by sultim-t** [comment]
What is the model of that Nvidia GPU?

**Comment by kaderci-johannes** [comment]
It says "NVIDIA GeForce GTX 1050 Ti" on GeForce Experience.

**Reply by Zosu** [reply]
Your GPU does not have hardware raytracing support. You could technically play some RTX games on the 10 series of cards through brute force compute but I can guarantee you that if you could get this running, it would not be playable for you.

**Reply by kaderci-johannes** [reply]
Thank you very much for the reply. It's sad that it's impossible on my hardware, as I would love the experience. What should I look for in a GPU if I want to play this mod?

**Reply by Zosu** [reply]
For the best experience with this mod you will want to have an Nvidia RTX GPU. I haven't looked closely at the performance myself yet, but I would say that you will want at least an RTX 3060 or faster class of GPU.

**Reply by kaderci-johannes** [reply]
Wow! That's quite resource intensive compared to HL1, which the original doesn't even need a 3D card to run. I may decide to play it after my (hopefully) next upgrade many years later. Thank you very much for your response and have a good rest of your weekend.
